notebooks/234-blip/Untitled.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>Not sure if this is review glitch or common for all, but images look very large to me relatively to text. I would suggest to reduce them somehow.</p><p>Also, few mistakes in text:</p><p>"Inasmuch", "texyt-based"</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/866/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>